_includes/homepage/homepage-info-left.html

```diff
@@ -18,7 +18,15 @@ <h1 class="has-text-secondary padding--bottom">
                 </a>
             </div>
             <div class="col is-half">
-                <img src="{{- site.baseurl -}}{{- section.section-image-path -}}" />
+                <img src="{{- site.baseurl -}}{{- section.section-image-path -}}" alt="
+                    {%- if section.section-image-alt -%}
```

Comment:
Could you explain why we're doing an if/elif/else? Shouldn't they be individual ifs?

Also, I think we should make the alt text mandatory if there is an image. So perhaps we could do 
```
{%- if section.section-image-alt -%}
  {{- section.section-image-alt -}}
{%- else -%}
  Please provide an alternative-text for the above image to improve accessibility for persons using screen readers
{%- endif -%}
```